aws/data_source_aws_ami_test.go

```diff
@@ -76,7 +75,6 @@ func TestAccAWSAmiDataSource_windowsInstance(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "image_type", "machine"),
 					resource.TestCheckResourceAttr(resourceName, "most_recent", "true"),
 					resource.TestMatchResourceAttr(resourceName, "name", regexp.MustCompile("^Windows_Server-2012-R2")),
-					resource.TestCheckResourceAttr(resourceName, "owner_id", "801119661308"),
```

Comment:
Unnecessary since `image_owner_alias` is checked to be `amazon`. Also, not consistent across partitions and potentially over time.